WireAnalytics/Sources/WireAnalytics/CountlyAbstraction/CountlyAbstraction.swift

```diff
@@ -16,11 +16,11 @@
 // along with this program. If not, see http://www.gnu.org/licenses/.
 //
 
-import Foundation
+public import Foundation
 
 // sourcery: AutoMockable
-/// Abstraction around `Countly`.
-protocol CountlyProtocol {
+/// Mirrors the API of `Countly`.
+public protocol CountlyAbstraction {
```

Comment:
Wasn't necessary. But in my opinion the term abstraction fits better. It was even in the protocol doc comment.
Let's hear the opinion of the second reviewer.

Reply:
No strong opinion here. I would probably have kept with `Protocol`, just because that is what we generally use so far for similar objects. We could also use `Interface`. 🤯

Reply:
Reverted! 306c0596fb97fde2a49d3d619de70ea4c98314c6